components/chunking/chunk-db-write-impl/src/main/kotlin/net/corda/chunking/db/impl/validation/ValidationFunctions.kt

```diff
@@ -133,16 +132,14 @@ fun CpiPersistence.persistCpiToDatabase(
                 cpkDbChangeLogEntities
             )
         } else {
-            throw ValidationException(
-                "CPI has already been inserted with cpks for " +
-                    "${cpi.metadata.cpiId.name} ${cpi.metadata.cpiId.version} with groupId=$groupId",
-                requestId
+            throw UnsupportedOperationException(
+                "CPI ${cpi.metadata.cpiId.name} ${cpi.metadata.cpiId.version} ${cpi.metadata.cpiId.signerSummaryHashForDbQuery}" +
```

Comment:
```suggestion
                "CPI ${cpi.metadata.cpiId.name} ${cpi.metadata.cpiId.version} ${cpi.metadata.cpiId.signerSummaryHashForDbQuery} " +
```

Space between parameter and "already exists..."